src/@next/Banner/Banner.tsx

```diff
@@ -123,7 +123,11 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
             <Icon name={iconNameValue} />
           </StyledFixedIconWrapper>
         )}
-        <StyledFixedBannerContentContainer data-noicon={showIcon ? '' : 'true'}>
+        <StyledFixedBannerContentContainer
+          data-noicon={showIcon ? '' : 'true'}
```

Comment:
I think we don't need ternary operator here, just 

`data-noicon={!showIcon}`

or better yet can change to

`data-show-icon={showIcon}`

Applicable to other data attributes

Reply:
I think it only applies for boolean right? will change the one for `showIcon`, but the others idt it's possible without using ternary operator, since they are not boolean values

Reply:
@marthayaputra done

Reply:
@michac789 I think you missed out this one 🤔 
`
data-has-action={action || secondaryAction ? 'true' : ''}`

Reply:
Maybe I'll make it to:
`!!(action || secondaryAction)`

Thought that since it is not boolean I should use ternary, but I think I can just use the snippet above

Reply:
changed @marthayaputra